home/managers.py

```diff
@@ -37,10 +37,10 @@ def with_applications(self, user):
         if user.is_anonymous:
             return self.annotate(completed_application=Value(False))
         return self.annotate(
-            completed_application=Exists(
+            completed_application=Subquery(
```

Comment:
Where is this change being used?

Reply:
It's used in `home/templates/home/includes/session_apply_btn.html` to have the id in order to create the link to the survey submission